### PR TITLE
Feature/pan mobile

### PR DESCRIPTION
**Pull request by monibinmile**

### CODE REVIEW

File: assets/css/newstyle.css [hunk 1]
```diff
@@ -442,14 +1194,17 @@ label.form-label {
...
+
 /* @media  (min-width: 425px) and (max-width: 767px) {
     .mobile{
         display: block !important;
     }
     .bankDetailsLeftSide{
         display: none !important ;
     }
-  } */
+  } */
```
Comment (abhisheksrivastav2) [inline assets/css/newstyle.css:1210]: @monibinmile Please add a new line here.

File: pan_mobile.html [hunk 2]
```diff
@@ -0,0 +1,402 @@
...
+                startTimer(59, timerElement);
+            });
+            startTimer(59, timerElement);
+        });
+    </script>
+</body>
+
+</html>
+
+</html>
```
Comment (abhisheksrivastav2) [inline pan_mobile.html:402]: Please add a empty line here also.

File: pan_mobile.html [hunk 3]
```diff
@@ -0,0 +1,402 @@
...
+            }
+
+            function showResendButton() {
+                resultElement.style.display = "block";
+                otpMsgElement.style.display = "none";
+            }
+
+            // Add event listener to the "Resend" button
+            const resendButton = document.getElementById("resend_otp");
+            const otp_msg_res = document.querySelector(".otp_msg"); // Add the period to select by class name
```
Comment (abhisheksrivastav2) [inline pan_mobile.html:388]: Is this comment left intended or is it safe to remove the comment?
If yes, then please remove the comment.

File: pan_mobile.html [hunk 4]
```diff
@@ -0,0 +1,402 @@
...
+                    minutes = parseInt(timer / 60, 10);
+                    seconds = parseInt(timer % 60, 10);
+
+
+                    minutes = minutes < 10 ? "0" + minutes : minutes;
+                    seconds = seconds < 10 ? "0" + seconds : seconds;
+
+
+                    display.textContent = minutes + ":" + seconds;
+
```
Comment (abhisheksrivastav2) [inline pan_mobile.html:359]: Please remove the empty lines, if not required.

File: assets/css/newstyle.css [hunk 5]
```diff
@@ -1,113 +1,866 @@
...
+.newsubmitbtn_first {
+    display: flex;
+    justify-content: space-between;
+    width: 304px;
+    padding: 12px;
+    align-items: center;
+    color: #fff;
+    background-color: #e31837;
+    border-color: #e31837;
+    /* box-shadow: #dc3545 0px 3px 18px; */
```
Comment (abhisheksrivastav2) [inline assets/css/newstyle.css:823]: Is this required? Can we remove this commented code?

**Review by abhisheksrivastav2 — COMMENTED** [review]
Please squash all your commits

File: pan_mobile.html (hunk 3 above)
Comment (monibinmile) [reply pan_mobile.html]: remove

**Pull request merged**